src/Core/Field.h

```diff
@@ -50,6 +50,7 @@ struct X : public FieldVector \
 
 DEFINE_FIELD_VECTOR(Array);
 DEFINE_FIELD_VECTOR(Tuple);
+DEFINE_FIELD_VECTOR(Map);
```

Comment:
```suggestion
// An array of [key1, value1, key2, value2, ...]
DEFINE_FIELD_VECTOR(Map);
```

Reply:
But it is `[(key1, value1), (key2, value2), ...]`

Reply:
> But it is [(key1, value1), (key2, value2), ...]

Proves the need for a comment :laughing: